### PR TITLE
Added ISSUE_TEMPLATES

### LINKED ISSUE #65: Having two issue templates to maintain the project
**Actual Behaviour**
The project only has one issue template to 

- report a bug 
- suggest a feature

this it's hard to identify or group issues depending on it's purpose or the content 

**Expected Behaviour**
Since the project is in the development stage it would be great to have two issue templates to maintain issues related to bug reporting or feature requests.

So, when a particular contributor wants to create a issue, once he/she press on the new issue but it will ask/popup the contributor to select the issue type (either bug reporting or feature suggest). Depending on the issue type they can create issues.

below attached is a screenshot of expected issue types

screen shot 2019-03-02 at 2 16 50 am

**Steps to reproduce it**

1. By creating two issue template types to report bugs or suggest features from project settings.

**Would you like to work on the issue?**
Since only the maintainers/contributors of the project has access to create new issue templates, I think one of them have to add it to the project

### PR DESCRIPTION
**Pull request by 0x7FFF**
Fixes #65 

**Changes**: Added ISSUE_TEMPLATES

**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

### CODE REVIEW

**Comment by adityastic** [comment]
This is the third time opening the pull request for the same issue

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+**LogCat for the issue**
+<!--Provide logs for the crash here.-->
+
+**Screenshots of the issue**
+<!--Where-ever possible add a screenshot of the issue.-->
+
+-   Device: (e.g. iPhone6)
+-   OS: (e.g. iOS8.1)
+-   Flutter version: 
+
```
Comment (adityastic) [inline .github/ISSUE_TEMPLATE/bug.md:29]: Add section for Additional Context about the Bug

**Review by yashk2000 — CHANGES_REQUESTED** [review]
Add a section for chores also. Take reference from the templates in other FOSSASIA repositories.

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (0x7FFF) [reply .github/ISSUE_TEMPLATE/bug.md:29]: Yes, my bad. I did some code changes in previous pull requests, in this one just specific issue fix as needed.

**Comment by 0x7FFF** [comment]
@yashk2000 @adityastic I think it should be good now

**Review by mariobehling — APPROVED** [review]

**Pull request merged**